### ISSUE TITLE
which F1 is the reported one?

### ISSUE THREAD

**Issue by mengyahuUSTC-PU** [ParticleMedia/RAGTruth#3]
Hi,

In your paper, you only reported one F1. If you use https://scikit-learn.org/stable/modules/generated/sklearn.metrics.classification_report.html, you would have F1 for label 0 , and label 1,  micro avg F1, macro avg F1. Could you let me know which F1 did you report?

**Comment by thuwyh** [comment]
We reported the F1 score for label 1(hallucination) in the paper.
